### PR TITLE
Add Feature: #293 Add implementation to TextProcessorJDK.topLetters()…

### LINKED ISSUE #293: Add implementation to TextProcessorJDK.topLetters() for haiku-kata-solutions module
The method `topLetters()` in the `TextProcessorJDK` class for the [haiku-kata-solutions](https://github.com/eclipse/eclipse-collections-kata/tree/52070f7651667c27e974e45eea2cce4309345530/haiku-kata-solutions) module is not yet implemented. Follow the TODOs and hints given in the [method](https://github.com/eclipse/eclipse-collections-kata/blob/52070f7651667c27e974e45eea2cce4309345530/haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorJDK.java#L37). Please do not add solutions to the non-solution modules. After writing the code, uncomment the test and tag annotations and check that the [test case](https://github.com/eclipse/eclipse-collections-kata/blob/52070f7651667c27e974e45eea2cce4309345530/haiku-kata-solutions/src/test/java/org/eclipse/collections/haikukata/TextProcessorJDKTest.java#L26) `topLetters()` in the `TextProcessorJDKTest` passes for the [haiku-kata-solutions](https://github.com/eclipse/eclipse-collections-kata/tree/52070f7651667c27e974e45eea2cce4309345530/haiku-kata-solutions) module. Make sure to adhere to our [contribution guide](https://github.com/eclipse/eclipse-collections-kata/blob/52070f7651667c27e974e45eea2cce4309345530/CONTRIBUTING.md). 

Please note that this issue is created for GHC 2023 Open Source Day, thus priority will be given to the attendees of this event. After the event, this issue will be open to anyone.

### PR DESCRIPTION
**Pull request by sakshi35**
… for haiku-kata-solutions module
Closes #293 Add implementation to TextProcessorJDK.topLetters()…

### CODE REVIEW

**Review by Sweta716 — APPROVED** [review]

**Comment by prathasirisha** [comment]
@sakshi35  can you please pull the latest changes and rebase on master. To achieve this, you can run the following commands -

git pull --rebase upstream master 
git push --force-with-lease

**Comment by sakshi35** [comment]
@prathasirisha have updated the branch

**Review by rzrobin213 — APPROVED** [review]

File: haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorJDK.java [hunk 1]
```diff
@@ -40,13 +41,16 @@ public List<Map.Entry<Character, Long>> topLetters()
...
+                .mapToObj(i -> Character.toLowerCase((char) i)).collect(Collectors.groupingBy(c -> c, Collectors.counting()));
 
         // TODO: Sort the entries in the Map by their values in reverseOrder
         // TODO: Take the top three entries and convert them to a List
         // Hint: Look at entrySet, stream, sorted, Map.Entry.comparingByValue, Comparator.reverseOrder()
         // Hint: On Stream look at limit and toList.
-        return null;
+
+
+        return chars.entrySet().stream().sorted(Map.Entry.comparingByValue(Comparator.reverseOrder())).limit(3).collect(Collectors.toList());
```
Comment (rachanavishwanath) [inline haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorJDK.java:53]: Please assign this result to a variable before returning to enhance code readability

**Comment by emilie-robichaud** [comment]
LGTM

**Pull request merged**